test/EventStore.Client.Tests.Common/GlobalEnvironment.cs

```diff
@@ -24,7 +23,6 @@ public static class GlobalEnvironment {
 
 		public static IDictionary<string, string> EnvironmentVariables(IDictionary<string, string>? overrides = null) {
 			var env = new Dictionary<string, string> {
-				[ImageTagName] = ImageTag,
```

Comment:
I think we need this line (unless it was removed intentionally?) for the cluster tests to know which tag to use if it isn't set as an environment var already